lib/vsc/install/shared_setup.py

```diff
@@ -756,7 +756,6 @@ def action_target(target, setupfn=setup, extra_sdist=[], urltemplate=None):
         'version': VERSION,
         'author': [sdw, ag, jt],
         'maintainer': [sdw, ag, jt],
-        'zip_safe': True,
```

Comment:
it's not an exception o something like that, it's broken everywhere. if people wonder why we ever remeoved it, the commit message looks clear enough.
if we ever manage to work around it, we can probably enable the zif_safe as default